openfisca_aotearoa/parameters/benefits/rates_rebates/income_threshold.yaml

```diff
@@ -0,0 +1,50 @@
+description: Income threshold
```

Comment:
It would be worth adding some more context, as this information is quite readable from the file name itself. Income threshold of what? For what? Help your reusers make sure you are indeed talking about the same thing: something along the lines of “Maximum yearly income to be entitled for rates rebate, combined for all owners and their spouses”.